build/install_python_deps.sh

```diff
@@ -5,7 +5,7 @@ set -e
 NEUROPOD_PYTHON_BINARY="python${NEUROPOD_PYTHON_VERSION}"
 
 # Install pip
-wget https://bootstrap.pypa.io/get-pip.py -O /tmp/get-pip.py
+wget https://bootstrap.pypa.io/2.7/get-pip.py -O /tmp/get-pip.py
```

Comment:
1. I see it is used at build/ci/install_lint_deps.sh too. 
2. Does it make sense to cache it? like:
```
tools/lint.sh-    if [ ! -f "/tmp/buildifier" ]; then
tools/lint.sh:        wget https://github.com/bazelbuild/buildtools/releases/download/2.2.1/buildifier -O /tmp/buildifier
tools/lint.sh-        chmod +x /tmp/buildifier
tools/lint.sh-    fi
```

or it is not used often?

Reply:
Not used often. The reason I built the caching in the lint script above (and some other places) is because it's used a lot